src/landing.tsx

```diff
@@ -0,0 +1,213 @@
+            <span className="je-brand">Jupyter Everywhere</span>
+          </h1>
+
+          <div className="je-buttons">
+            <div
+              className="je-card"
+              onClick={() => handleCreateNotebook('python')}
+              onKeyDown={e => handleCardKeyDown(e, 'python')}
+              tabIndex={0}
+              role="button"
+              aria-label="Create Python Notebook"
+            >
+              <p>Create Python Notebook</p>
+              <img src={pythonLogo} alt="Python logo" />
+            </div>
+
+            <div
+              className="je-card"
+              onClick={() => handleCreateNotebook('xr')}
+              onKeyDown={e => handleCardKeyDown(e, 'xr')}
+              tabIndex={0}
+              role="button"
+              aria-label="Create R Notebook"
+            >
+              <p>Create R Notebook</p>
+              <img src={rLogo} alt="R logo" />
+            </div>
+          </div>
+          <a
+            href="#"
+            className="je-upload"
+            onClick={e => {
+              e.preventDefault();
+              handleUpload();
+            }}
+          >
+            Upload a notebook
+          </a>
+        </main>
+
+        <div className="je-hero-bottom">
+          <a href="#features" className="je-scroll-indicator" onClick={handleScrollToFeatures}>
+            Scroll to learn more
+            <br />
+            <img src={scrolldownArrow} alt="An arrow that scrolls down when clicked" />
+          </a>
+        </div>
+      </section>
+
+      {/* Features section */}
+      <section id="features" className="je-features">
+        <h2>Features</h2>
+        <div className="je-features-grid">
+          <div className="je-feature">
+            <img src={codeSymbol} alt="Code symbol" />
+            <h3>Code Ready</h3>
+            <p>
+              You can choose between the Python and R languages to create classroom friendly
+              interactive notebooks
+            </p>
+          </div>
+
+          <div className="je-feature">
+            <img src={shareIcon} alt="Share icon" />
+            <h3>Shareable</h3>
+            <p>
+              You can easily generate a shareable link that will allow you to share your work with
+              others or come back to it later
+            </p>
+          </div>
+
+          <div className="je-feature">
+            <img src={jupyterLogo} alt="Jupyter logo" />
+            <h3>Jupyter Compatible</h3>
+            <p>Notebooks are compatible with the JupyterLab and Jupyter notebooks ecosystems</p>
+          </div>
+        </div>
+      </section>
+
+      {/* Testimonial section */}
+      <section className="je-testimonial">
+        <img src={testimonialAvatar} alt="User avatar" className="je-avatar" />
+        <blockquote>
+          "This application makes it so much easier for us to focus on the lesson and not on
+          technical issues. I have already recommended it to others."
```

Comment:
To match design 1-1:

```suggestion
          “This application makes it so much easier for us to focus on the lesson and not on
          technical issues. I have already recommended it to others”
```